RecipeMatch/Views/GridRecipeCell.m

```diff
@@ -9,11 +9,19 @@
 #import "UIKit+AFNetworking.h"
 
 @implementation GridRecipeCell
+static const float CORNER_RADIUS = 15;
+
 // clears image and title of cell
 - (void)prepareForReuse {
     [super prepareForReuse];
     [self.imageView cancelImageDownloadTask];
```

Comment:
might as well just use the sdWebImage version for canceling `sd_cancelCurrentImageLoad`